runtimes.yml

```diff
@@ -0,0 +1,59 @@
+version: 0.1
+
+runtimes:
+  java:
+    versions:
+      corretto17:
+        commands:
+          - echo "Installing Java version 17 ..."
+
+          - export JAVA_HOME="$JAVA_17_HOME"
+
+          - export JRE_HOME="$JRE_17_HOME"
+
+          - export JDK_HOME="$JDK_17_HOME"
+
+          - |-
+            for tool_path in "$JAVA_HOME"/bin/*;
+             do tool=`basename "$tool_path"`;
+              if [ $tool != 'java-rmi.cgi' ];
+              then
+               update-alternatives --list "$tool" | grep -q "$tool_path" \
+                && update-alternatives --set "$tool" "$tool_path";
+              fi;
+            done
+  golang:
+    versions:
+      1.18:
+        commands:
+          - echo "Installing Go version 1.18 ..."
+          - goenv global  $GOLANG_18_VERSION
+  python:
+    versions:
+      3.8:
+        commands:
+          - echo "Installing Python version 3.8 ..."
+          - pyenv global  $PYTHON_38_VERSION
+  php:
```

Comment:
what are we using PHP for? Is this just something that was in codebuild docker images? I have a feeling that this is safe to remove